<launch>

   <arg name="ns" default="f450"/>

   <arg name="rgb_img_topic" default="/camera/color/image_raw"/>
   <arg name="pcloud_topic"  default="/camera/depth_registered/points"/>

   <arg name="colors_json" default="$(find bricks_detection)/cfg/hsv_colors.json"/>
   
   <group ns="$(arg ns)">
      <node name="bricks_detection" pkg="bricks_detection" type="bricks_detection_node" output="screen" respawn="false">
         <param name="rgb_img_topic" type="str" value="$(arg rgb_img_topic)"/>
         <param name="pcloud_topic"  type="str" value="$(arg pcloud_topic)"/>
         <param name="colors_json"   type="str" value="$(arg colors_json)"/>
      </node>
   </group>

</launch>